src/Components/__tests__/Utils/addressForm.ts

```diff
@@ -58,3 +60,25 @@ export const fillAddressForm = (component: any, address: Address) => {
     value: address.phoneNumber!,
   })
 }
+
+export const fillAddressFormTL = (address: Address) => {
+  const name = screen.getByPlaceholderText("Full name")
+  const country = screen.getByRole("combobox")
+  const addressLine1 = screen.getByPlaceholderText("Street address")
+  const addressLine2 = screen.getByPlaceholderText("Apt, floor, suite, etc.")
+  const city = screen.getByPlaceholderText("City")
+  const region = screen.getByPlaceholderText("State, province, or region")
+  const postalCode = screen.getByPlaceholderText("ZIP/postal code")
+  const phoneNumber = screen.getAllByPlaceholderText(
+    "Add phone number including country code"
+  )[0]
+
+  userEvent.paste(name, address.name)
```

Comment:
TL can be slow, and it's a tradeoff we will decide. Here is an example I changed to `paste` from `type` (which is significantly slower). In this case we don't really care about each key stroke so I think it's acceptable. If we do need to consider each key, such as autocomplete use cases, we might need to accept the slowness.

See [a5782ec](https://github.com/artsy/force/pull/12867/commits/a5782ec99010954c22cfc6d2b5abeff0632e601e) for more details.